### PR TITLE
Sprite

### PR DESCRIPTION
**Pull request by simonVansuyt**
I have add a sprite for a the share bar icon.

### CODE REVIEW

**Comment by arthurvr** [comment]
Seems good, but I would prefer to auto-generate those sprites using grunt. Also, the according css must be included automatically. This kind of spriting is difficult to maintain.

**Comment by simonVansuyt** [comment]
Odjoe mulle dikke homo

File: src/stylus/header.styl [hunk 1]
```diff
@@ -80,42 +80,43 @@
...
 						&.facebook {
-							background: url(../images/icon-facebook.png) no-repeat center;
-							background-size: contain;
+							background-position: -149px 0;
+							background-size: cover;
 						}
 						&.blog {
-							background: url(../images/icon-blog.png) no-repeat center;
-							background-size: contain;
+              background-position: -4px 0;
```
Comment (SrPatinhas) [inline src/stylus/header.styl:102]: fix this identation please

**Comment by simonVansuyt** [comment]
Odjoe mulle dikke homo

**Comment by frdmn** [comment]
@simonVansuyt english, please!

**Comment by simonVansuyt** [comment]
Sorry but thats was to arthurvr

**Comment by arthurvr** [comment]
Anyhow, do it in english ;-)

**Comment by kingflurkel** [comment]
simon, zidde gij nen belg?

**Comment by simonVansuyt** [comment]
No but arthurvr is a Belgian

**Comment by kingflurkel** [comment]
cool. 

antwerp represent anyway

**Comment by simonVansuyt** [comment]
Can somebody fix this

**Comment by ghost** [comment]
Its going to be merged into the grunt file.

**Pull request closed without merging**